manager/integration/tests/test_upgrade.py

```diff
@@ -307,4 +358,29 @@ def test_upgrade(longhorn_upgrade_type,
     res_pod_md5sum = get_pod_data_md5sum(core_api, pod_name, pod_data_path)
     assert res_pod_md5sum == pod_md5sum
 
-    check_volume_data(volume1, volume1_data)
+    check_volume_data(volume0, volume0_data_after_sys_upgrade)
+    check_volume_data(volume1, volume1_data_after_sys_upgrade)
+    check_volume_data(volume2, volume2_data_after_sys_upgrade)
+
+    # Delete one healthy replica for volume2 to trigger the rebuilding
```

Comment:
Is it possible to use existing functions? ex:
- [delete_replica_on_test_node](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L1967)
- [wait_for_rebuild_start](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L4005)
- [wait_for_rebuild_complete](https://github.com/longhorn/longhorn-tests/blob/master/manager/integration/tests/common.py#L4005)